installation.md

```diff
@@ -25,7 +25,7 @@ It's recommended to install Python using [Homebrew](https://brew.sh), however in
 
 Because of Linux having many different package managers, we've decided to make the way to install on Linux via `pip`, or building from source using `python` and `poetry`, which isn't covered here.
 
-1. Install Python 3.8 or higher (3.11 recommended) from your package manager - for example `sudo apt install python3` or `sudo pacman -S python3`
+1. Install Python 3.8 or higher (3.11 recommended) from your package manager - for example `sudo apt install python3` or `sudo pacman -S python3` or `sudo dnf install python3` 
```

Comment:
What's that?

Reply:
This is fedora's default packaging command for installing python3.

Reply:
Well then put the distro names in parentheses (to make it useful) or remove the last two examples altogether (to only serve as an example).

Reply:
On it.

Reply:
Not the best i could do in commit but feel free to send suggestions on how to make it better if needed.